src/renderer/components/reader/Reader.tsx

```diff
@@ -182,15 +180,16 @@ interface ReaderState {
     menuOpen: boolean;
     fullscreen: boolean;
     indexes: any;
-    visibleBookmarkList: Locator[];
+    visibleBookmarkList: any[];
+    currentLocation: any;
```

Comment:
Instead of the `any` type, could this be `currentLocation: LocatorExtended | undefined;`?
(with `import { LocatorExtended } from "@r2-navigator-js/electron/renderer/index";`)